components/molecules/RadioButtons.js

```diff
@@ -0,0 +1,16 @@
+import styles from '../../styles/RadioButtons.module.css';
+
+export default function RadioButtons(props) {
+
+    return (
+        <div className={styles.container}>
+            <p>{props.label}</p>
+            {props.items.map((item) =>
+                <div key={item.short}>
+                    <input type="radio" id={item.short} defaultChecked={props.state.get == item.short ? true : false} value={item.short} name={props.id}/>
```

Comment:
not 100% familiar with react, but you shouldn't need to declare true/false, just let the expression do that

```suggestion
                    <input type="radio" id={item.short} defaultChecked={props.state.get == item.short} value={item.short} name={props.id}/>
```

Reply:
True, good catch!